docs/pages/repo/docs/handbook/dev.mdx

```diff
@@ -112,6 +112,36 @@ For example:
 turbo run dev --filter docs
 ```
 
+## Running setup tasks
+
+There may be some tasks that you need to run _before_ your persistent, long-running development tasks are running. Some examples of setup steps would be:
+
+- Pre-building a package
+- Setting up Docker containers
+- Checking Node or package manager versions
+
+If you name these setup tasks `dev`, you'll have a naming collision in `turbo.json` with your persistent tasks. Instead, you can rename those setup tasks to `pre-dev` and depend on those `pre-dev` tasks for your `dev` tasks:
+
+```json filename="turbo.json"
+{
+  "pipeline": {
+    "dev": {
+      "dependsOn": ["^pre-dev"],
```

Comment:
Should we add a note here about when you may just want `"dependsOn": ["pre-dev"],` instead?